hera_cal/abscal.py

```diff
@@ -1446,28 +1450,42 @@ def abscal_run(data_file, model_files, refant=None, calfits_infile=None, verbose
         return return_obj
 
 
-def cut_bls(datacontainer, bls, bl_cut):
+def cut_bls(datacontainer, bls, min_bl_cut=None, max_bl_cut=None, inplace=False):
     """
-    Cut visibility data based on maximum baseline length. Note
-    that this directly overwrites the data in these containers (i.e. inplace).
+    Cut visibility data based on min and max baseline length.
 
     Parameters
     ----------
     datacontainer : DataContainer object to perform baseline cut on
 
-    bls : dictionary, keys are antenna-pair tuples and values are baseline vectors in meters
+    bls : dictionary, holding baseline position vectors.
+        keys are antenna-pair tuples and values are baseline vectors in meters
+
+    min_bl_cut : float, minimum baseline separation [meters] to keep in data
+
+    max_bl_cut : float, maximum baseline separation [meters] to keep in data    
 
-    bl_cut : float, maximum baseline separation [meters] to keep in data
+    inplace : bool, if True edit data in input object, else make a copy.
 
-    Output (cut_datacontainers)
+    Output
     ------
-    cut_datacontainer : DataContainer object with bl cut enacted
+    datacontainer : DataContainer object with bl cut enacted
```

Comment:
I think it's generally better to keep the returns of the same form whenever possible... so in this case, I'd recommend returning the cut datacontainer regardless. If people want to use inplace, it's OK to have a return and just call it without any assignment operator.

Reply:
sure thing